packages/cloud_security_posture/manifest.yml

```
@@ -1,15 +1,16 @@
 format_version: 1.0.0
 name: cloud_security_posture
 title: "Kubernetes Security Posture Management"
-version: 0.0.24
+version: 0.0.26
+release: beta
 license: basic
 description: "Check Kubernetes cluster compliance with the Kubernetes CIS benchmark."
 type: integration
 categories:
   - containers
   - kubernetes
 conditions:
-  kibana.version: "^8.5.0"
+  kibana.version: "^8.4.0"
```

This has to be 
```suggestion
  kibana.version: "^8.5.0"
```
no?

---

Sure, overrode by upstream merge.